cgrok/grok_capture.c

```diff
@@ -8,6 +8,50 @@
 
 char *EMPTYSTR = "";
 
+/* 
+ * public domain strtok_r() by Charlie Gordon
```

Comment:
I've never heard explicitly that public domain is an acceptable license (I'm pretty sure it is). We need to ticket this + get confirmation asap. If we wait to long we may end up blocking the release on getting a lawyer to +1 (this has happened before unfortunately).